src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 10;
+    private int currentSize = 0;
```

Comment:
```suggestion
    private int currentSize;
```
0 - дефолтне значення для змінної типу int